### PR TITLE
Replace infinite scroll with paging in transactions list

### PR DESCRIPTION
**Pull request by fvictorio**
This PR replaces the infinite scroll with normal paging in the transactions list page.

I added only the "First", "Older" and "Newer" buttons. To have a "Last" button we would also need to have a precise number of transactions instead of an estimate. I suppose an estimate is used because doing a full count is too expensive, so I'm not sure what can we do here. A possible option is to have a statistics table that keeps track of how many transactions are. This table could be kept on sync with a trigger on inserts to the transactions table.

### CODE REVIEW

**Comment by brwatanabe** [comment]
refers to #1101

File: apps/block_scout_web/assets/js/pages/transactions.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import humps from 'humps'
 import numeral from 'numeral'
 import socket from '../socket'
 import { createStore, connectElements } from '../lib/redux_helpers.js'
-import { withInfiniteScroll, connectInfiniteScroll } from '../lib/infinite_scroll_helpers'
+import { withInfiniteScroll } from '../lib/infinite_scroll_helpers'
```
Comment (GuSilverFlame) [inline apps/block_scout_web/assets/js/pages/transactions.js]: is the `withInfiniteScroll` function still necessary here even without the infinite scroll?

**Comment by feliperenan** [comment]
I suggest just remove the infinite scroll changing the code back to how it was before - with only the older button  - in this PR.

My concern is that we don't know how to implement the previous button yet, at least we didn't refine it.   Once we have the older button back, we can think about how to implement the previous button and how it will look like in a different PR.

cc @acravenho

**Comment by fvictorio** [comment]
Closes in favor of #1137.

**Pull request closed without merging**